openapi/platform/platform.yaml

```diff
@@ -4352,10 +4456,20 @@ components:
       type: object
       description: |-
         [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
-        
+
         An Automation is a trigger and set of actions performed when the trigger is activated.
       x-tags:
         - Automations
+      x-examples:
+        example-1:
+          type: automations
+          id: Automation-91cb048c-c498-43ba-ab99-2ca2d27d2c46
+          attributes:
+            name: Example Automation
+            trigger: /app/account-group/account-group/created
+            status: 'on'
+            tags:
+              - '["example-tag"]'
```

Comment:
For most cases the example can be auto generated based on the example values you specified for each individual field. That allows you to delete this manually specified example so you don't have to maintain it.